src/utils/PageEmbed.ts

```diff
@@ -328,7 +330,7 @@ export class PageEmbed extends EmitterBase
                         return;
                     }
 
-                    msg.delete();
+                    await msg.delete();
```

Comment:
I didn't await because it takes a long time and I don't need to

Reply:
true, i awaited cuz i thought it was causing the problem

Reply:
h

Reply:
removed all the awaits. should be good i think